hazelcast/src/main/java/com/hazelcast/topic/TopicOverloadPolicy.java

```diff
@@ -46,7 +46,7 @@
     DISCARD_OLDEST,
 
     /**
-     * The message that was to be published, is discarded.
+     * The message that was to be published, is discarded when there is no space in the ringbuffer.
```

Comment:
All of the policies deal with the situation when there is no space in the ringbuffer.